cloud-formation/src/templates/state-machine.yaml

```diff
@@ -63,7 +76,6 @@ States:
       is created and the old is terminated.
     End: True
 
-  FailureHandler:
-    Type: Task
-    Resource: "${FailureHandlerLambda.Arn}"
-    Next: WaitForBrowserPoll
\ No newline at end of file
+  FailState:
```

Comment:
So presumably this causes the overall State Machine Execution status to be failed, which means that the status endpoint in support-frontend ([here?](https://github.com/guardian/support-frontend/blob/5d8b40e37cfb1509583b2fbd9b82ced3f6d468fc/app/services/stepfunctions/RegularContributionsClient.scala#L120)) will return a failure status?

Reply:
Yes